src/components/nodes/pool/pool.vue

```diff
@@ -182,6 +182,9 @@ export default {
     createLaneSet() {
       const laneSet = this.moddle.create('bpmn:LaneSet');
       this.laneSet = laneSet;
+      const generator = this.$parent.nodeIdGenerator;
```

Comment:
Instead of using `this.$parent` could you please pass the generator in as a prop?